catroid/src/main/java/org/catrobat/catroid/devices/arduino/ArduinoListener.java

```diff
@@ -171,4 +174,12 @@ public int getPortValue(int pin) {
 	public void setPortValue(int pin, int value) {
 		this.portValue[pin] = value;
 	}
+
+	public int getuCPortValue(int port) {
+		return uCPortValue[port];
+	}
+
+	public void setuCPortValue(int port, int value) {
+		this.uCPortValue[port] = value;
+	}
```

Comment:
Is this method really necessary? I also doubt there should be a setter for `setPortValue`.
The port values should only be set in the `onAnalogMessageReceived`, right?